epa_hr_expense_tier_validation/models/hr_expense_sheet.py

```diff
@@ -0,0 +1,12 @@
+# Copyright 2024 - TODAY, Kaynnan Lemes <kaynnan.lemes@escodoo.com.br>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class HrExpenseSheet(models.Model):
+
+    _inherit = "hr.expense.sheet"
+    # Adapting state according to v12.0 usability
+    _state_from = ["draft"]
+    _state_to = ["submit", "approve", "post", "done"]
```

Comment:
isso nao viria de algum modulo de tier validation do hr_expense ? isso pelo jeito nao eh uma customização e sim um modulo novo ?

Reply:
Ele é uma customização do hr_expenses_tier_validation eu estou sobrescrevendo o state from e state to para que seja igual ao da v12.0

Reply:
Na versão 14 o tier.validation somente ocorre quando esta em submit, então a custom seria para ocorrer em draft @marcelsavegnago

Reply:
A mudança seria na linha 10 e 11 para sobrescrever https://github.com/OCA/hr-expense/blob/14.0/hr_expense_tier_validation/models/hr_expense_sheet.py#L10

Reply:
Subi um novo commit movendo o módulo para incluir a nomenclatura de custom, não tinha me atentado ao nome, o da branch ficou correto mas o módulo em si ficou faltando o _custom hehe